docs/api_specs/index.html.md

```diff
@@ -22,7 +22,8 @@ search: true
 
 # Introduction
 
-This is the HTTP-RPC API for the ChildChain and Watcher.
+This is the HTTP-RPC API for the Child Chain Server and Watcher.
 
 All calls use HTTP POST and pass options in the request body in JSON format.
-Errors will usually return with HTTP response code 200, and the details of the error in the response body. See [Errors](#errors).
+Errors will usually return with HTTP response code 200, and the details of the error in the response body.
```

Comment:
maybe we should change HTTP code when error occurs.
409 (CONFLICT) probably in the majority will fit

Reply:
I think this has been debated and decided to play along the `eWallet` HTTP-RPC rules, refer to @pnowosie for more background